src/protocol.h

```diff
@@ -263,7 +263,7 @@ enum ServiceFlags : uint64_t {
     NODE_BLOOM = (1 << 2),
     // NODE_WITNESS indicates that a node can be asked for blocks and transactions including
     // witness data.
-    NODE_WITNESS = (1 << 3),
```

Comment:
I think the fact that the names of these constants are being used by  explorers and tools to render service flags is enough reason to use a different name (NODE_WITNESS2X), to ensure consistency.